### ISSUE TITLE
parameters of lsm

### ISSUE THREAD

**Issue by shirleyatgithub** [IGITUGraz/LSM#4]
Dear author,
Thanks for your generous sharing. Your code is very helpful for me to understand lsm.
I just wonder how you set the parameters to be the values they are now.

**Comment by anandtrex** [comment]
Hi @shirleyatgithub , thank you for your message.  The current parameters were chosen using a parameter search to achieve good performance on the XOR and memory tasks while maintaining reasonable firing rates and being close to asynchronous irregular firing.

**Comment by shirleyatgithub** [comment]
Dear author,
Thanks for the reply. I wonder do you have any paper related to this project? I would like to read.

**Comment by shirleyatgithub** [comment]
Dear author,
I have a question about the input_spikes. I see that the input_states are 666 pair of 0 and 1, and that corresponds to the targets (666 logits). But why the inp_spikes is 3226 elements with two array. How should I understand their meaning?
Thank you very much.

**Comment by anandtrex** [comment]
Hi @shirleyatgithub, this model was used in [this paper](https://iopscience.iop.org/article/10.1088/1748-3190/aa7663/meta). 

Regarding your second question, can you point me to the specific line number where this happens?
